### PR TITLE
Add container tags endpoint

### PR DESCRIPTION
**Pull request by newswangerd**
Adds `/api/galaxy/_ui/v1/execution-environments/<repository>/_content/tags/` endpoint that lists the tags in a container repository along with the container manifest that each tag corresponds to.

### Filters
- name
  - icontains
  - exact
  - starts_with
  - contains
- tagged_manifest__digest
  - icontains
  - exact
  - starts_with
  - contains

### Sort Options
- pulp_created
- name

### Sample Response
```
{
    "meta": {
        "count": 1
    },
    "links": {
        "first": "/api/automation-hub/_ui/v1/execution-environments/repositories/alpine/_content/tags/?limit=10&offset=0&tagged_manifest__digest__icontains=sha256%3Abe9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c",
        "previous": null,
        "next": null,
        "last": "/api/automation-hub/_ui/v1/execution-environments/repositories/alpine/_content/tags/?limit=10&offset=0&tagged_manifest__digest__icontains=sha256%3Abe9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c"
    },
    "data": [
        {
            "name": "latest",
            "pulp_created": "2021-09-10T19:20:18.260593Z",
            "pulp_last_updated": "2021-09-10T19:20:18.260632Z",
            "tagged_manifest": {
                "pulp_id": "69197e2c-d124-4a3b-aaa8-b3d81bc51b95",
                "digest": "sha256:be9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c",
                "schema_version": 2,
                "media_type": "application/vnd.docker.distribution.manifest.v2+json",
                "pulp_created": "2021-09-10T19:20:18.248274Z"
            }
        }
    ]
}
```

### CODE REVIEW

File: galaxy_ng/app/api/ui/viewsets/execution_environment.py [hunk 1]
```diff
@@ -97,6 +115,18 @@ def get_distro(self):
             models.ContainerDistribution, base_path=self.kwargs["base_path"])
 
 
+class ContainerTagViewset(ContainerContentBaseViewset):
+    permission_classes = [access_policy.ContainerRepositoryAccessPolicy]
+    serializer_class = serializers.ContainerTagSerializer
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = TagFilter
+
+    def get_queryset(self):
```
Comment (rochacbruno) [inline galaxy_ng/app/api/ui/viewsets/execution_environment.py:124]: Line 75 said 
```
# Tag filters are supported, but are done in get_queryset. See the comment
        # there
```

is it supposed to have a comment here?

Comment (newswangerd) [reply galaxy_ng/app/api/ui/viewsets/execution_environment.py:124]: Ah, that refers to a different endpoint. Let me clarify the comments.

**Comment by himdel** [comment]
using `?sort=pulp_created` leads to a 400, `sort=created` works though, not sure if intentional?

**Review by himdel — APPROVED** [review]
Works for me :+1: 

(tested sort, filtering by name icontains, digest exact and paging)

**Pull request merged**